provider/lxd/export_test.go

```diff
@@ -70,11 +70,11 @@ func ExposeInstEnv(inst *environInstance) *environ {
 }
 
 func ExposeEnvConfig(env *environ) *environConfig {
-	return env.ecfg
+	return env.ecfgUnlocked
```

Comment:
Why not `Config()`? You use the `server()` below.

Reply:
because Config() returns *config.Config not *environConfig